jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpMethod.java

```diff
@@ -95,7 +95,7 @@ public byte[] getBytes()
 
     public boolean is(String s)
     {
-        return toString().equalsIgnoreCase(s);
+        return toString().equals(s);
```

Comment:
I think this is more spec-compliant, but this is a behavioral change from versions 9.4/10/11. I'm worried this could cause issues with users unknowingly relying on this case-insensitive comparison down the road.

Reply:
Actually I don't think this actually does change behaviour in the common cases.  We always lookup a method in the cache (even when in case insensitive mode) and replace the string with the one we find.  Thus any code that does `HttpMethod.FOOis(request.getMethod())` will see the "normalized" case method name.     Thus I'm comfortable making this change and dealing with any consequences.